data/json/weather_type.json

```
@@ -269,6 +269,76 @@
       ]
     }
   },
+  {
+    "id": "mist",
+    "type": "weather_type",
+    "name": "Mist",
+    "color": "light_gray",
+    "map_color": "dark_gray_white",
+    "sym": "~",
+    "ranged_penalty": 3,
+    "sight_penalty": 1.2,
+    "light_modifier": -20,
+    "sun_multiplier": 0.3,
+    "sound_attn": 0,
+    "dangerous": false,
+    "precip": "light",
+    "rains": false,
+    "sound_category": "silent",
+    "priority": 40,
+    "condition": {
+      "and": [
+        { "math": [ "u_val( 'weather:windpower' )", ">", "12" ] },
+        { "math": [ "u_val( 'weather:windpower' )", "<", "0" ] },
```

Isn't it supposed to be like this?
```suggestion
        { "math": [ "u_val( 'weather:windpower' )", "<", "12" ] },
        { "math": [ "u_val( 'weather:windpower' )", ">", "0" ] },
```

---

oh god, i really inversed the wind values and miss it